views/Hangouts.tsx

```diff
@@ -81,7 +82,10 @@ class Hangouts extends React.Component<Props> {
       <SafeAreaView style={styles.userList}>
         <UserList />
         <Modal
-          isVisible={this.state.modalVisible || this.props.popupModal}
+          isVisible={
+            (this.state.modalVisible || this.props.popupModal) &&
+            !this.props.isReviewing
```

Comment:
I have added the extra boolean `isReviewing` to be able to control whether or not this model is rendered with an action from a child component. This might not be the best way to handle it, but I couldn't figure out how to do it and keep our existing functionality. We need to be able to close this modal in order to show the modal for the review.